gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/EntityServiceAndGatewayServiceConverter.java

```diff
@@ -158,28 +159,19 @@ public static LiteralConstant convertToQueryLiteral(
         .build();
   }
 
+  private static String longToIso(org.hypertrace.gateway.service.v1.common.Expression expression) {
```

Comment:
naming nit: literalSecondsToIsoString . A long isn't implicitly in seconds - let's be explicit, and also make sure that matches existing assumptions.